src/app/contribute/guidelines/component-guidelines/index.html

```diff
@@ -0,0 +1,269 @@
+<stache
```

Comment:
I suggest we rename this directory to `components`, so the URL will be `/guidelines/components`.

Reply:
✔️